### DISCUSSION TITLE
about "Zombie child" described in ReadMe

### DISCUSSION THREAD

**Discussion by fongfai** [pmndrs/zustand#1504] [Q&A] [answered]
My question is what aspects in zustand iteself to deal with this "zombie child" problem  except for in  #302 discussed which use react's unstable_batchedUpdates to solve, thanks. 
![image](https://user-images.githubusercontent.com/19684413/210207932-1aacdc18-104a-4cd5-b99b-1f33e9229080.png)

**Comment by dai-shi** [comment, marked as answer]
It's mostly about history of development (I suspect in v1 and v2). I think we can delete the entire line now.

**Reply by fongfai** [reply]
> It's mostly about history of development (I suspect in v1 and v2). I think we can delete the entire line now.

got it, thanks

**Reply by fongfai** [reply]
for the "react concurrency" problem there is an "use-sync-external-store" API to deal with;   and,  could you show the key point to resolve ”context loss“ or  no need context fundamentally?

**Reply by dai-shi** [reply]
I explained those in this podcast: https://podcast.20minjs.com/1952066/11541711
Let me know if it's not very clear.

**Reply by fongfai** [reply]
> I explained those in this podcast: https://podcast.20minjs.com/1952066/11541711 Let me know if it's not very clear.

thanks for your podcast link,  I learnd from that you mentioned 'context loss'(https://github.com/facebook/react/issues/13332) like multi portal(sorry, I just got this message from your talk);  so, I want to know is that your point is zustand  solved this problem's way  is because zustand itself no need context at all?

**Reply by dai-shi** [reply]
yeah, that's what I understand. at the point of the first release of zustand, there are very few libs that don't use context, so it was a selling point. though, nowadays, there are many such libs.
